site/en_AU.all.json

```diff
@@ -524,7 +524,7 @@
     "one": "Automatically renews every {{.Interval}}",
     "other": "Automatically renews every {{.Count}} {{.Interval}}s"
   },
-  "plans_page_header": { "other": "Available Plans" },
+  "plans_page_header": { "other": "Passes" },
```

Comment:
I think we should say "Plans" or "Available Plans" here and let them override it using a feature toggle or lang file

Reply:
- Reverted to Available Plans